### PR TITLE
Added Venue message support

### PR DESCRIPTION
**Pull request by vlad-lukyanov**
by calling SendVenue(recipient Recipient, venue *Venue, options *SendOptions)

### CODE REVIEW

**Comment by tucnak** [comment]
Would you please be able to resolve the conflicts, so I could merge it in or should I do it on my own?

**Comment by vlad-lukyanov** [comment]
@tucnak , done.

File: types.go [hunk 1]
```diff
@@ -176,3 +176,11 @@ type Callback struct {
...
 	Data      string `json:"data"`
 }
+
+// Venue object represents a venue location with name, address and optional foursquare id.
+type Venue struct {
+	Location Location		`json:"location"`
+	Title string			`json:"title"`
+	Address string			`json:"address"`
+	Foursquare_id string	`json:"foursquare_id",omitempty`
+}
```
Comment (zoni) [inline types.go]: It would be nice if this was `gofmt`'ed :smile:

**Comment by vlad-lukyanov** [comment]
@zoni , done)

**Pull request merged**